config/initializers/assets.rb

```diff
@@ -4,7 +4,7 @@
 Rails.application.config.assets.version = '1.0'
 
 # Add additional assets to the asset load path
-# Rails.application.config.assets.paths << Emoji.images_path
+Rails.application.config.assets.paths << Rails.root.join("app", "assets", "images", "icons")
```

Comment:
Is this not done automatically?  Or should we refer to `icons/<icon name>` everywhere?